src/commands/commandConstants.ts

```diff
@@ -85,4 +90,9 @@ export namespace ServerCommandConstants {
    * Command to associate a grammar in a XML document
    */
   export const ASSOCIATE_GRAMMAR_INSERT = "xml.associate.grammar.insert";
+
+  /**
+   * Command to check if the current XML document is bound to a grammar
+   */
+  export const CHECK_BOUND_GRAMMAR = "xml.check.bound.grammar"

```

Comment:
I would rename to 

`export const ASSOCIATE_GRAMMAR_CHECK = "xml.associate.grammar.check";`